frontend/src/features/leases/detail/LeasePages/insurance/details/Insurance.tsx

```diff
@@ -0,0 +1,44 @@
+import { FormSection } from 'components/common/form/styles';
+import { IInsurance } from 'interfaces';
+import { Col, Row } from 'react-bootstrap';
+import { ILookupCode } from 'store/slices/lookupCodes';
+
+import Policy from './Policy';
+
+export interface InsuranceDetailsViewProps {
+  insuranceList: IInsurance[];
+  insuranceTypes: ILookupCode[];
+}
+
+const InsuranceDetailsView: React.FunctionComponent<InsuranceDetailsViewProps> = ({
+  insuranceList,
+  insuranceTypes,
+}) => {
+  return (
+    <>
+      {insuranceList
```

Comment:
worth noting that this sort will run whenever the user interacts with the form, ie. every time they type a character.

Reply:
yikes. I think for now it works fine since those are only 5 categories